### PR TITLE
feat: publish dagger-py-functions.zip to github packages

### PR DESCRIPTION
**Pull request by MayurGubrele**

### CODE REVIEW

File: dagger-py-functions/udfs/scalar/sample_udf.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+from pyflink.table.udf import udf
+
+class SampleUdf(ScalarFunction):
+
+    def __init__(self):
+        self.counter = None
+
+    def open(self, function_context):
+        mg = function_context.get_metric_group()
+        self.counter = mg.add_group("udf", "SampleUdf").counter("value")
```
Comment (prakharmathur82) [inline dagger-py-functions/udfs/scalar/sample_udf.py:11]: what exactly do we want to publish to metrics?

File: build.gradle [hunk 2]
```diff
@@ -1,7 +1,25 @@
...
+
+plugins {
+    id 'maven-publish'
+}
+
 ext {
     flinkVersion = System.getenv('flinkVersion') ?: '1.14.3'
 }
 
+def pyZipVersion = '0.0.1'
```
Comment (prakharmathur82) [inline build.gradle:20]: this should come from version.txt

File: dagger-py-functions/udfs/scalar/sample_udf.py (hunk 1 above)
Comment (MayurGubrele) [reply dagger-py-functions/udfs/scalar/sample_udf.py]: removed the metric part from the code.

File: build.gradle (hunk 2 above)
Comment (MayurGubrele) [reply build.gradle]: Changed pyZipVersion to take value from version.txt

**Pull request merged**